data/json/recipes/recipe_electronics.json

```diff
@@ -1600,18 +1600,17 @@
   "skill_used": "fabrication",
   "skills_required" : [ "electronics", 2 ],
   "difficulty": 5,
-  "time": 40000,
+  "time": 3000,
   "autolearn": true,
+  "charges": 1,
   "qualities" : [
-    {"id":"SAW_M","level":1},
-    {"id":"CUT","level":1},
-    {"id":"HAMMER","level":2}
+    { "id": "ANVIL", "level": 1 },
+    { "id": "HAMMER", "level": 2 },
+    { "id": "CHISEL", "level": 3 },
+    { "id": "CUT", "level": 1 }
   ],
-  "tools": [ [ [ "boltcutters", -1 ], [ "toolset", -1 ] ] ],
-  "components": [
-    [ [ "copper", 100 ], [ "scrap_copper", 1 ] ],
-    [ [ "duct_tape", 100 ] ]
-  ]
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "surface_heat", 1, "LIST" ], [ "forge", 1 ], [ "oxy_torch", 1 ] ] ],
```

Comment:
```suggestion
    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "surface_heat", 1, "LIST" ], [ "forging_standard", 1, "LIST" ] ] ],
```

Reply:
I didn't do that because this requirement uses 20 charges at once, while in this recipe only 1 is needed, to keep it same as other heating tools. Should I make a new requirement then?

Reply:
No. But why only 1 charge? What good can 1 charge do?

Reply:
It's per one "unit" of wire. If you wanted to make a whole spool (200 units), it'd be 200 charges and that's quite a lot.

Reply:
Ok, got it.